package.json

```diff
@@ -16,9 +16,10 @@
     "clean": "yarn workspaces foreach -vp run clean",
     "test": "jest",
     "test:strict": "USE_STRICT=1 jest",
+    "test-storybook": "test-storybook",
+    "test-storybook:pull-main": "test-storybook --url https://pixiv.github.io/charcoal --updateSnapshot",
```

Comment:
Is this command never used?

Reply:
yes
I thought it would be nice to provide it as an option, so I deleted it.